skin-resque-frontend/src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,33 @@
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import Heading from '../Heading/Heading';
+import './Pagination.scss';
+
+export default function Pagination({ color, size }) {
+
+	let pages = [];
+	for (let i = 1; i <= size; i++) {
+		pages.push(i);
+	};
+
+	const pagesList = pages.map((pageNumber) => {
```

Comment:
Instead use `pages.map((_, index) => bodyOfFunction)`. Use index instead of array of incrementing numbers.